clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -780,6 +837,7 @@ template <typename Derived> class SyclKernelFieldHandler {
   virtual void handleStructType(FieldDecl *, QualType) {}
   virtual void handleReferenceType(FieldDecl *, QualType) {}
   virtual void handlePointerType(FieldDecl *, QualType) {}
+  virtual void handleArrayType(const CXXBaseSpecifier &, QualType) {}
```

Comment:
How can someone inherit from an array?

Reply:
Base class support does not exist. This artifact will be deleted.